include/seqan3/alignment/matrix/trace_directions.hpp

```diff
@@ -47,3 +47,21 @@ namespace seqan3
 template <>
 constexpr bool add_enum_bitwise_operators<seqan3::detail::trace_directions> = true;
 } // namespace seqan3
+
+#include <seqan3/io/stream/debug_stream.hpp>
```

Comment:
move include to the top?

Reply:
This is currently not working, because of cyclic inclusion.

`trace_direction` includes `debug_stream` includes `alignment_matrix_formatter` includes `trace_direction`

A follow up PR, where I remove `alignment_matrix_formatter` allows me to fix this problem.

Reply:
I feel like there are on average two follow ups for each PR you do :see_no_evil: I know small PRs are easier to review but this way it is hard to keep track of what you want to achieve on the long run.